resources/assets/less/bem/beatmap-scoreboard-table.less

```diff
@@ -20,10 +20,27 @@
   @_top: beatmap-scoreboard-table;
   @row-height: 1.8em;
 
-
   width: 100%;
   overflow-x: auto;
 
+  .row-colour(@colour) {
+    td {
```

Comment:
not sure how much it helps but `> td` is probably slightly faster (and doesn't randomly recolor the nonexistent child tables).